spec/src/main/asciidoc/microprofile-opentracing.asciidoc

```diff
@@ -168,10 +168,13 @@ Tracing in `javax.ws.rs.client.Client` has to be explicitly enabled by invoking
 The implementation might enable client tracing globally, in this case explicit configuration has no effect.
 
 ===== MicroProfile Rest Client
-Tracing for MicroProfile Rest Client is by default globally enabled and it can be disabled by
+Tracing for this client is by default globally enabled and it can be disabled by
 specifying `@Traced(false)` on the client interface or method. When it is specified on the client's
 interface tracing is disabled for all methods.
 
+Note that integration with MicroProfile Rest Client is not mandatory for vendors not implementing
```

Comment:
@fmhwong @pilhuhn could you please have a look? I am adding this to clarify a parial support if the client spec is not supported by the vendor.\

Reply:
We do not mention any version numbers in readme, I think it's ok to go without the version for the consistency. However I will add the version to the changelog at least.